modules/web/client/src/main/scala/observe/ui/components/sequence/SequenceTable.scala

```diff
@@ -227,7 +227,10 @@ private sealed trait SequenceTableBuilder[S: Eq, D <: DynamicConfig: Eq]
                 sequence.indexWhere(row => CurrentHeadersRowIds.contains_(getRowId(row).value)) - 1,
                 ScrollOptions
               )
-      .render: (props, resize, cols, _, _, _, table, _, virtualizerRef) =>
+      .useMemoBy((_, _, _, visits, _, _, _, _, _) => visits)((_, _, _, _, _, _, _, _, _) =>
```

Comment:
Let's use a regular `val` for the `id` set.

By using `useMemo` we are comparing the whole list of `Visit`s on each rerender, which is probably more expensive than just building the `id` set.

Other than this, still LGTM.

Reply:
Done, I'll merge this now